src/app/drivingLicence/controllers/check-your-details.js

```diff
@@ -4,6 +4,10 @@ const logger = require("hmpo-logger").get();
 class CheckYourDetailsController extends BaseController {
   async saveValues(req, res, next) {
     try {
+      req.sessionModel.set(
```

Comment:
Could this be set in the root controller with the other fields?

Reply:
This is now in the root controller with the other fields